locale/en/get-involved/node-speakers.md

```diff
@@ -141,6 +141,12 @@ If anyone is reported as malicious or making others feel uncomfortable, they may
 - [Github](https://github.com/MarkPieszak)
 - Topics - Node.js, JavaScript SSR (Server-side Rendering), Performance, Open Source, Community
 
+###### Julián Duque 🇨🇴
```

Comment:
Could you please remove the flag after your name for consistency ?

We only add flag after country name :)

```suggestion
###### Julián Duque
```